### DISCUSSION TITLE
Release v1.0.0

### DISCUSSION THREAD

**Discussion by yunjjeongjo** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#260] [04.Relases Note]
### 🏠  아트집(Art.zip) - Release v1.0.0  🏠
# 페이지 별 기본 요구사항
> ## 로그인 페이지
- 사용자는 로그인을 할 수 있다.
- 사용자는 소셜 로그인(카카오)을 할 수 있다.
> ## 회원가입 페이지
- 사용자는 회원가입을 할 수 있습니다.
- 유효성 검사
   - 이메일 : 이메일 형식인지 검사합니다.
   - 닉네임 : 공백문제를 제외하고 10글자 이내인지 검사합니다. 
   - 비밀번호 :  영문, 숫자, 특수문자를 포함해 8-13인지 검사합니다.
   - 비밀번호 확인 : 비밀번호와 일치하는지 검사합니다.
> ## 메인 페이지
- 사용자는 스와이퍼를 통해 다가오는 전시회와 인기많은 전시회를 확인할 수 있다.
- 사용자는 현재 자신의 위치를 기반으로 3km 이내의 전시회들을 확인할 수 있다.
> ## 전시회 상세 페이지
- 사용자는 전시회 상세정보를 확인할 수 있다.
- 사용자 해당 전시회에 좋아요를 누를 수 있다.
- 사용자는 아이콘 클릭을 통해 전시회 주소를 클립보드에 복사 할 수 있다.
- 사용자는 카카오 지도를 통해 해당 전시회의 위치를 확인할 수 있다.
- 사용자는 인기순(좋아요 갯수)으로 정렬된 리뷰 4개를 미리보기 할 수 있다.
> ## 더보기 페이지
- 사용자는 더 많은 다가오는 전시회와 더 많은 인기많은 전시회를 페이지네이션으로 확인할 수 있다.
> ## 검색결과 페이지
- 사용자는 원하는 전시회 제목을 검색하여 해당하는 검색결과를 페이지네이션으로 확인할 수 있다.
> ## 후기 작성 & 수정 페이지 
- 로그인한 사용자는 후기를 작성해서 등록할 수 있다.
   - 다녀 온 전시회를 검색할 수 있다.
   - 다녀 온 날짜를 등록할 수 있다.
   - 제목과 내용을 텍스트로 작성할 수 있다.
   - 사진을 여러 개 업로드할 수 있다.
   - 공개 여부를 설정할 수 있다.
- 로그인한 사용자는 등록한 후기를 수정할 수 있다.
> ## 유저 페이지
case 1 - 자신(로그인 한 유저)의 페이지
case 2 - 다른 유저의 페이지
![image](https://user-images.githubusercontent.com/93373357/188809510-3844ace4-1431-4641-be44-993608b3e74a.png)
> ## 커뮤니티 페이지
- 사용자는 최근에 작성된 후기들을 피드 형식으로 볼 수 있다.
- 사용자는 무한 스크롤을 통하여 더 많은 후기를 불러올 수 있다.
- 사용자는 피드를 클릭하면 후기 상세 페이지로 이동할 수 있다.
- 사용자는 피드를 통하여 좋아요를 누를 수 있다.
- 사용자는 피드의 게시글이 본인의 게시글인 경우, 게시글을 수정하거나 삭제할 수 있다.
> ## 후기 상세 페이지
- 사용자는 후기의 상세 내용을 조회할 수 있다.
- 사용자는 후기에 좋아요를 남길 수 있다.
- 사용자는 후기의 댓글을 조회할 수 있다.
- 사용자는 무한 스크롤을 통하여 더 많은 댓글을 불러올 수 있다.
- 사용자는 후기에 댓글을 남길 수 있다.
- 사용자는 댓글에 답글을 남길 수 있다
- 사용자는 더보기 버튼을 통하여 더 많은 답글을 불러올 수 있다.
- 사용자는 후기의 댓글이나 답글이 자신의 댓글일 경우, 수정하거나 삭제할 수 있다.
- 사용자는 댓글이나 답글에 좋아요를 남길 수 있다.
> ## 맞춤전시회 페이지

<hr /><em>This discussion was created from the release <a href='https://github.com/prgrms-web-devcourse/Team-BackFro-ArtZip-FE/releases/tag/v1.0.0'>Release v1.0.0 </a>.</em>

**Comment by yunjjeongjo** [comment]
@Hyevvy 맞춤전시회 혜경님 담당 페이지 맞죠 ?? 기억이 가물가물 ,,
페이지별 기능을 위키를 기반으로 작성했는데 맞춤전시회 페이지는 위키에 내용이 없더라구요!
적어주시면 감사하겠습니다 ~

**Comment by gitul0515** [comment]
수고하셨습니다 👍
